### PR TITLE
Allow for easier override of BaseVocabularyView settings

### PR DESCRIPTION
**Pull request by Gagaro**
I need to override some variable in `BaseVocabularyView` but they aren't easily accessible as of right now. This solves it.

### CODE REVIEW

File: plone/app/content/browser/vocabulary.py [hunk 1]
```diff
@@ -74,6 +74,50 @@ class VocabLookupException(Exception):
 
 class BaseVocabularyView(BrowserView):
 
+    def get_translated_ignored(self):
+        return [
+            'author_name',
```
Comment (jensens) [inline plone/app/content/browser/vocabulary.py]: while at it, it is probably a good idea to make the default list a module global variable like so:

    TRANSLATED_IGNORE = [
        'author_name',
        'cmf_uid',
        'commentators',
        'created',
        ...
    ]
    ...

        def get_translated_ignored(self):
            return TRANSLATED_IGNORE

**Comment by Gagaro** [comment]
Test running.

**Comment by Gagaro** [comment]
The test failing seems to be timeout and I have an error when I try to build the test again.

**Comment by Gagaro** [comment]
This branch has no conflict, what is the `35 needs: rebase` label for?

**Comment by jensens** [comment]
@Gagaro tests are now running with one failure
http://jenkins.plone.org/job/pull-request-5.1/1367/testReport/junit/plone.app.content.tests.test_widgets/BrowserTest/testVocabularyEncoding/

**Comment by Gagaro** [comment]
I rebased on master, the test passed locally so it should be good, waiting on travis now.

**Comment by jensens** [comment]
@Gagaro this is Jenkins here, so it does not start on its own, you need to do it. I paste the standard explanation how to do so below.

----

You can always start a Jenkins test-job on your own to test that your pull request (PR) does not break anything. 

Below click on the link to Jenkins
screenshot from 2016-07-25 11 20 06

**log in** with your Github user - if not already done before.

Paste the full **pull request URL** into the text field (if multiple pull requests need to be combined in one run, then one PR URL per line)

Click on **Build**

Jenkins reports back to the pull request(s) on Github in the status box at the bottom.

In Plone we value a lot our testing suite, so we usually never merge pull requests if they have not been tested by Jenkins first.

**Comment by Gagaro** [comment]
Ho yes indeed I forgot it was manual. Sorry about that.

**Comment by Gagaro** [comment]
tests are green

**Pull request merged**